docs/data-sources/database.md

```diff
@@ -0,0 +1,107 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "encore_database Data Source - terraform-provider-encore"
+subcategory: ""
+description: |-
+  Encore provisioned database information
+---
+
+# encore_database (Data Source)
+
+Encore provisioned database information
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of the Encore resource
+
+### Optional
+
+- `env` (String) The environment of the Encore resource. Defaults to the provider environment
+
+### Read-Only
+
+- `aws_rds` (Attributes) (see [below for nested schema](#nestedatt--aws_rds))
```

Comment:
I guess these belong to the cluster, but maybe adding additional indirection is just annoying

Reply:
yeah, I was thinking the same thing. We don't really add anything related to the database so felt weird to put it in a cluster sub object.

Reply:
I guess we want to add the cloud name for the database, in case it's different from the encore name